<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditPracticWork</class>
 <widget class="QDialog" name="EditPracticWork">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>598</width>
    <height>162</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Корректировка лабораторной работы - RayTracing</string>
  </property>
  <widget class="QPushButton" name="button_exit">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>100</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Закрыть окно добавления новой работы</string>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: rgb(255, 255, 255);
    border: 2px solid #094065;
}

QPushButton:pressed {
    background-color:  rgb(118, 118, 118);
}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>images/exit-button.png</normaloff>images/exit-button.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>36</width>
     <height>36</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="label_name_work">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>20</y>
     <width>341</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Путь к файлу:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Лабораторная работа: </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="text_path">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>60</y>
     <width>381</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="button_help">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Получить справку</string>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: rgb(255, 255, 255);
    border: 2px solid #094065;
}

QPushButton:pressed {
    background-color:  rgb(118, 118, 118);
}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>images/help-button.png</normaloff>images/help-button.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>36</width>
     <height>36</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="button_edit">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>100</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Сохранить и отправить работу</string>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: rgb(255, 255, 255);
    border: 2px solid #094065;
}

QPushButton:pressed {
    background-color:  rgb(118, 118, 118);
}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>images/complete-button.png</normaloff>images/complete-button.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>36</width>
     <height>36</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="button_path">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>60</y>
     <width>31</width>
     <height>22</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Указать путь к файлу</string>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: rgb(255, 255, 255);
    border: 2px solid #094065;
}

QPushButton:pressed {
    background-color:  rgb(118, 118, 118);
}
</string>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
